### ISSUE TITLE
Resource Type module does not list associated Resource Sub Types

### ISSUE THREAD

**Issue by daiClegg** [dbrownukk/EFD_v2#71]
In Resource Type module select list view.
Select any resource type (e.g. Employment - which has 12 Resource Sub Types in the test instance, or Use of Livestock, in which i just created 2 Resource Sub Types).

Note that the Resource SubType frame displays no records in the list, but in the bottom right-hand corner it correctly states the number of items that should be in the list:

![image](https://user-images.githubusercontent.com/30879677/36968727-4f33f7f2-205b-11e8-866d-4bb2db5fbb5e.png)

**Comment by dbrownukk** [comment]
`the defaults for the List were not set 

Go to List settings using the gear 
Pick the Ladder to add / remove columns
Reset using the restore default value

Strange that it was set - perhaps remove this ability to modify layout??

**Comment by daiClegg** [comment]
Tested Ok.

I have used the ability to add columns to display, but that was in the early days before some of the UX tweaks were made to better fit our known use cases, and i haven't found myself needing it recently. 

But if i were a researcher exploring the data in order to do an export say, i might well use it together with the export to Excel feature for quick and easy data extract.
So perhaps we keep it but don't tell anyone it's there?
